Go/1-simple_test.go

```diff
@@ -0,0 +1,15 @@
+package emitter
+
+import "fmt"
+
+func ExampleSimple() {
```

Comment:
@DzyubSpirit, you named the example incorrectly, it doesn't refer to the type it is an example of and thus, it won't be rendered by GoDoc correctly. I think the correct way to call this example here is `ExampleEventEmitter` according to the `testing` package [documentation](https://golang.org/pkg/testing/#hdr-Examples).